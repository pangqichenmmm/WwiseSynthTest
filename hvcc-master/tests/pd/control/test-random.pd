#N canvas 692 327 208 192 10;
#X obj 23 17 loadbang;
#X obj 23 134 random 10;
#X obj 23 157 print;
#X msg 62 96 seed 11;
#X obj 23 44 t a a a a a a a a;
#X msg 122 71 seed 7672;
#X connect 0 0 4 0;
#X connect 1 0 2 0;
#X connect 3 0 1 0;
#X connect 4 0 1 0;
#X connect 4 1 1 0;
#X connect 4 2 1 0;
#X connect 4 3 3 0;
#X connect 4 4 1 0;
#X connect 4 5 1 0;
#X connect 4 6 1 0;
#X connect 4 7 5 0;
#X connect 5 0 1 0;
