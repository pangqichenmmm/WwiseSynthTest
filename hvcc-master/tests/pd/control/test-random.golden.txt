[@ 0.000] print: 2
[@ 0.000] print: 7
[@ 0.000] print: 1
[@ 0.000] print: 7
[@ 0.000] print: 0
[@ 0.000] print: 5
